pkg/snapstore/oss_snapstore_test.go

```diff
@@ -151,3 +172,11 @@ func (m *mockOSSBucket) DeleteObject(objectKey string, _ ...oss.Option) error {
 	delete(m.objects, objectKey)
 	return nil
 }
+
+// GetBucketWorm get bucket worm configuration for given bucket name.
+func (m *mockOSSClient) GetBucketWorm(_ string, _ ...oss.Option) (oss.WormConfiguration, error) {
+	return oss.WormConfiguration{
+		State:                 "InProgress",
```

Comment:
Could you clarify the reasoning behind using a static "InProgress" state here?

Reply:
there are 3 states possible:
1. "Expired": if you have enable the retentionPolicy but didn't lock it then retentionPolicy will get expire.
2. "InProgress" :  if you have enable the retentionPolicy but neither it got locked nor retentionPolicy got expired.
3. "Locked":  if you have enable the retentionPolicy but it got locked.

So, for tests either we can use either `InProgress` or `Locked`, I don't have strong opinion which one to use.